control-plane/controllers/configentry_controller_ent_test.go

```diff
@@ -29,11 +29,323 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-// NOTE: We're not testing each controller type here because that's done in
+// NOTE: We're not testing each controller type here because that's mostly done in
 // the OSS tests and it would result in too many permutations. Instead
-// we're only testing with the ServiceDefaults and ProxyDefaults controller which will exercise
-// all the namespaces code for config entries that are namespaced and those that
+// we're only testing with the ServiceDefaults and ProxyDefaults controllers which
+// will exercise all the namespaces code for config entries that are namespaced and those that
 // exist in the global namespace.
+// We also test Enterprise only features like SamenessGroups.
+
+func TestConfigEntryController_createsConfigEntry(t *testing.T) {
```

Comment:
maybe we can change this from createsConfigEntry -> createsEntConfigEntry to show it's only running ent config entries.

Reply:
Same for update/delete cases as well